### PR TITLE
add YAML file to .github/workflows/ to use Github Actions for CI

### PR DESCRIPTION
**Pull request by gotanygrapes-lemonade**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
+[![tests](https://github.com/envato/packer-ami-copy-buildkite-plugin/actions/workflows/test.yml/badge.svg?branch=main)](https://github.com/envato/packer-ami-copy-buildkite-plugin/actions/workflows/test.yml)
```
Comment (orien) [inline README.md]: Let's move the badge below the document title.

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+    name: Lint
+    runs-on: ubuntu-latest
+    container:
+      image: buildkite/plugin-linter:latest
+      volumes:
+        - "${{github.workspace}}:/plugin"
+    steps:
+      - uses: actions/checkout@v2
+      - name: lint
+        run: lint --id envato/packer-ami-copy-buildkite-plugin
```
Comment (orien) [inline .github/workflows/tests.yml]:
```suggestion
        run: lint --id envato/packer-ami-copy
```

**Review by orien — COMMENTED** [review]
It looks like there's a bit of work getting the tests and Shellcheck to pass

**Comment by gotanygrapes-lemonade** [comment]
> It looks like there's a bit of work getting the tests and Shellcheck to pass

Thanks Orien, Will refer to the other repos to figure out how to get the tests pass

**Comment by orien** [comment]
I wonder if deleting this line helps:

https://github.com/envato/packer-ami-copy-buildkite-plugin/blob/2f0f214c3e7781b91b76bfbbc9b093ef36b5a498/tests/command.bats#L4

**Comment by orien** [comment]
Awesome! The tests are passing.

Now just the Shellcheck. We'll need to resolve the issues it finds:

> Error: ./hooks/command:53:12: error: Double quote array expansions to avoid re-splitting elements. [SC2068]
Error: ./hooks/command:62:11: error: Double quote array expansions to avoid re-splitting elements. [SC2068]
Warning: ./hooks/command:71:1: warning: Variable was used as an array but is now assigned a string. [SC2178]
Error: ./hooks/command:81:12: error: Double quote array expansions to avoid re-splitting elements. [SC2068]
Error: ./hooks/command:83:23: error: Double quote array expansions to avoid re-splitting elements. [SC2068]

**Comment by gotanygrapes-lemonade** [comment]
> Awesome! The tests are passing.
> 
> Now just the Shellcheck. We'll need to resolve the issues it finds:
> 
> > Error: ./hooks/command:53:12: error: Double quote array expansions to avoid re-splitting elements. [SC2068]
> > Error: ./hooks/command:62:11: error: Double quote array expansions to avoid re-splitting elements. [SC2068]
> > Warning: ./hooks/command:71:1: warning: Variable was used as an array but is now assigned a string. [SC2178]
> > Error: ./hooks/command:81:12: error: Double quote array expansions to avoid re-splitting elements. [SC2068]
> > Error: ./hooks/command:83:23: error: Double quote array expansions to avoid re-splitting elements. [SC2068]

I double quotes the array expansion and it no longer alerts the error in shellcheck, but it started to fail the tests/Tests... 

I'm still trying to figure out why it says it's assigned a string... It looks like an array to me > > Warning: ./hooks/command:71:1: warning: Variable was used as an array but is now assigned a string. [SC2178]

**Review by orien — APPROVED** [review]
👍

**Pull request merged**